### PR TITLE
57 feature progress bar function

### PR DESCRIPTION
**Pull request by VitorMeirelesOliveira**
## Description

In this issue, we worked to make a dynamic progress bar. A function was created and parameters can be sent to it.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

I don't want to.

### CODE REVIEW

**Comment by VitorMeirelesOliveira** [comment]
image
As you can see this screenshot shows that the progress bar is working fine.

File: app/scripts/contentscript.js [hunk 1]
```diff
@@ -191,6 +193,59 @@ const insertProgressBar = (activity, forgotten) => {
...
+        .appendChild(progressbar)
+    document.documentElement.style
+        .setProperty('--progressfunction', generic_rate);
+
+    //Create table to format the description
+    document.getElementById("barfunction").innerHTML = [
+    '<div style="float: justify">',
+    '<TABLE BORDER=0>',
+        '<TR>',
+        '<h1 id="text" style="text-align:center; font-size: 20px"></h1>',
```
Comment (filipetoyoshima) [inline app/scripts/contentscript.js]: On [prototype](https://cjjcastro.gitlab.io/2019-1-hubcare-docs/project/prototype/), the title is aligned on left, not on center.

**Comment by VitorMeirelesOliveira** [comment]
> Please test your modifications using jest-s

Could you, please, be more specific...

**Comment by vitorAlves7** [comment]
This space is from the commits div for weeks, it is created but the graph is not being inserted.

**Comment by filipetoyoshima** [comment]
> This space is from the commits div for weeks, it is created but the graph is not being inserted.

You're right! Sorry for the mistake.

**Pull request merged**